app/models/automation_request.rb

```diff
@@ -29,7 +29,7 @@ def self.create_from_ws(version, user, uri_parts, parameters, requester)
     options[:schedule_type] = parameters['schedule_time'].present? ? "schedule" : "immediately"
     options[:schedule_time] = parameters['schedule_time'].to_i.days.from_now if parameters['schedule_time']
 
-    object_parameters = parse_out_objects(parameters)
+    object_parameters, parameters = parse_out_objects(parameters)
```

Comment:
This section (lines 32-36) could also be isolated into a private methods, `build_attrs`, which is the only resulting object we care about.  Feel free to pick a better method name.